server/src/mocks/pay-gov-mocks.es6

```diff
@@ -6,91 +6,106 @@ const uuid = require('uuid/v4');
 const util = require('../services/util.es6');
 const treesDb = require('../models/trees-db.es6');
 const middleware = require('../services/middleware.es6');
-
+const paygov = require('../services/paygov');
 const templates = require('./pay-gov-templates.es6');
 
-const payGov = {};
+const mockPayGov = {};
 
 const transactions = {};
-const tokens = {};
 
-/** router for mock pay.gov  specific endpoints */
-payGov.router = express.Router();
+function isStartCollection(requestBody) {
```

Comment:
Refactor mock paygov to more accurately reflect the actual service and improve understandability. The mock interface can be used in a stateless way for unit testing and a stateful way for the dev environment.  Added a couple additional failure cases directly from the QA environment.